audio/rsgain/Portfile

```
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                      1.0
+
+PortGroup                       github 1.0
+PortGroup                       cmake 1.1
+
+github.setup                    complexlogic rsgain 3.5.2 v
+
+categories                      audio
+platforms                       darwin freebsd linux netbsd openbsd
+license                         GPL-2+
+
+maintainers                     {gmail.com:lmkrawiec @lmkra}
+
+description                     Really simple gain
+long_description                ReplayGain 2.0 command line utility \
+                                rsgain applies loudness metadata tags to your files, \
+                                while leaving the audio stream untouched. \
+                                A ReplayGain-compatible player will dynamically adjust \
+                                the volume of your tagged files during playback.
+
+homepage                        https://github.com/complexlogic/rsgain
+
+github.tarball_from             releases
+distname                        ${github.project}-${github.version}-source
+use_xz                          yes
+worksrcdir                      ${github.project}-${github.version}
+
+checksums                       rmd160  c4152e8b4d7534b82e0c1c15821bbf1241616f90 \
+                                sha256  4f701521fd270b60f536a12aeb64e09bb87f24c30d576231d2fda3b6cc8c697e \
+                                size    51204
+
+set libfmt_version              11
+
+depends_build                   port:cmake \
```

You're using the `cmake` portgroup, so you should not need to specify `cmake` in `depends_build`.

You may instead want to use `depends_build-append` because the `depends_build` list gets initially set by the `cmake` portgroup, and then `depends_build-append` will add onto it without overwriting it.

```suggestion
depends_build-append            port:pkgconfig
```